android/src/main/java/com/mapbox/mapboxgl/MapboxMapController.java

```diff
@@ -553,12 +553,45 @@ public void onError(@NonNull String message) {
         result.success(symbolId);
         break;
       }
+      case "symbol#add_many": {
```

Comment:
If we decide to go with the other two suggestions, we might be able to make this something like:
```java
List<String> newSymbolIds = null;
final List<Object> options = call.argument("options");
List<SymbolOptions> symbolOptionsList = new ArrayList<SymbolOptions>();
if (options != null) {
   SymbolBuilder symbolBuilder;
   for (Object o : options) {
      symbolBuilder = new SymbolBuilder();
      Convert.interpretSymbolOptions(o, symbolBuilder);
      symbolOptionsList.put(symbolBuilder.getSymbolOptions());
   }
   Map<String, SymbolController> newSymbolControllers = new LinkedHashMap<String, SymbolController>();
   if (!symbolOptionsList.isEmpty()) {
      List<Symbol> newSymbols = symbolManager.create(symbolOptionsList);
      String symbolId;
      for (Symbol symbol : newSymbols) {
         symbolId = String.valueOf(symbol.getId());
         newSymbolControllers.put(symbolId, new SymbolController(symbol, true, symbolTappedListener));
      }
      symbols.putAll(newSymbolControllers);
   }
   newSymbolIds = new ArrayList<>(newSymbolControllers.keySet());
}
result.success(newSymbolIds);
break;
```